Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:lts
```

```suggestion
FROM node:20.14-slim
```
o que acha de usar uma imagem menor? não sei se vale mantermos muito recurso no nosso container que não vai ser utilizado